### DISCUSSION TITLE
在创立者产品帖下评论即可有机会赢取MQ-99！

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#2876] [其他]
![屏幕截图 2024-08-18 233804](https://image.gcores.com/f0431133-53b8-4e4e-8b0a-5b20fbe97d92.png?x-oss-process=image/quality,q_90/format,webp/watermark,image_d2F0ZXJtYXJrLnBuZw,g_se,x_10,y_10)
# 即日起 在创立者工业产品帖下发布评论即可有机会赢取MQ-99小型AI无人机！
## 我们将抽取十名用户送出无人机
# *注 过于简短的无意义评论将不会视为参加
# *注 本帖下评论不视为参加
# *注 恶意诋毁我司的评论将不视为参加并且拉入我司黑名单
# 将于新艾利都时8月31号自动开奖

**Comment by oh-holy-mom** [comment]
我勒个8.32啊

**Reply by GrunderIndustries** [reply]
系临时工失误 已经修改

**Comment by nostalgiatan** [comment]
什么是创立者工业文章下？

**Reply by GrunderIndustries** [reply]
我们已经将撰写该文的临时工辞退 为您带来的不便敬请谅解
